safe_cli/operators/safe_tx_service_operator.py

```diff
@@ -406,3 +411,58 @@ def drain(self, to: ChecksumAddress):
             print_formatted_text(
                 HTML("<ansigreen>Safe account is currently empty</ansigreen>")
             )
+
+    def search_account(
+        self, address: ChecksumAddress
+    ) -> Optional[Union[LocalAccount, HwWallet]]:
+        """
+        Search the provided address between loaded owners
```

Comment:
Leave one blank line between explanation and params
```suggestion
        Search the provided address between loaded owners
        
```